flink-runtime/src/main/java/org/apache/flink/runtime/checkpoint/DefaultCheckpointStatsTracker.java

```diff
@@ -434,6 +434,9 @@ private void setDurationSpanAttribute(
     @VisibleForTesting
     static final String LATEST_COMPLETED_CHECKPOINT_ID_METRIC = "lastCompletedCheckpointId";
 
+    @VisibleForTesting
+    static final String LATEST_COMPLETED_CHECKPOINT_TIMESTAMP = "lastCheckpointCompletedTimestamp";
```

Comment:
I think we should rename to `LATEST_CHECKPOINT_COMPLETED_TIMESTAMP`

Reply:
done